BusyList/Handlers/AddHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BusyList.Commands;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusyList.Handlers
+{
+    public class AddHandler : IHandler<AddCommand>
+    {
+        private ITaskRepository _taskRepository;
+
+        public AddHandler(ITaskRepository taskRepository)
+        {
+            _taskRepository = taskRepository;
+        }
+        public void Run(AddCommand command)
+        {
+            AddTaskData item = new AddTaskData(command.Description);
+            _taskRepository.AddTask(item);
+
+            Console.WriteLine($"Added task with description:{item.Description}.");
```

Comment:
Perhaps let the user know what ID the task was assigned?

Reply:
I have no idea how to do that since the AddTaskData contains only the description.

Reply:
When adding a task to the db, the newly created task will be returned
So we send in a AddTaskData, then we recieve a TaskItem which does contain the id of the task